RELEASING/changelog.py

```diff
@@ -24,9 +24,9 @@
 from click.core import Context
 
 try:
-    from github import BadCredentialsException, Github, PullRequest, Repository
+    from github import BadCredentialsException, GitHub, PullRequest, Repository
 except ModuleNotFoundError:
-    print("PyGithub is a required package for this script")
+    print("PyGitHub is a required package for this script")
     exit(1)
```

Comment:
If these are third party modules, then these changes will break. It's trivial for me to drop individual change families although I'll probably try to carefully tease out the text and noncode instances of GitHub.

Reply:
can you check that this change actually works by installing pygithub and trying to import that class?

Reply:
Tested. 

Apparently the project does at least occasionally call itself **PyGitHub**:
https://github.com/PyGithub/PyGithub/blob/001970d4a828017f704f6744a5775b4207a6523c/README.md?plain=1#L1

But, the classes/exceptions/methods aren't exposed via the proper spelling, so I'll have to fix up this commit (dropping most of the Python bits, but keeping the change for line 29).

---

As an aside, I can't get `pip install -r ...` to work, it dies on my m1 trying to do stuff using numpy (amd64 stuff at that!!):

```
            numpy/core/src/multiarray/scalartypes.c.src:2967:65: error: too few arguments to function call, expected 2, have 1
                return _Py_HashDouble((double) PyArrayScalar_VAL(obj, Float));
                       ~~~~~~~~~~~~~~                                       ^
```
I've tried a couple of versions of numpy and that didn't help.